Sources/Datadog/CrashReporting/DDCrashReportingPluginType.swift

```diff
@@ -10,14 +10,140 @@ import Foundation
+        /// A client-generated 16-byte UUID of the incident.
+        internal let incidentIdentifier: String?
+        /// The name of the crashed process.
+        internal let processName: String?
+        /// Parent process information.
+        internal let parentProcess: String?
+        /// The location of the executable on disk.
+        internal let path: String?
+        /// The CPU architecture of the process that crashed.
+        internal let codeType: String?
+        /// The name of the corresponding BSD termination signal.
+        internal let exceptionType: String?
+        /// CPU specific information about the exception encoded into 64-bit hexadecimal number preceded by the signal code.
+        internal let exceptionCodes: String?
+
+        public init(
+            incidentIdentifier: String?,
+            processName: String?,
+            parentProcess: String?,
+            path: String?,
+            codeType: String?,
+            exceptionType: String?,
+            exceptionCodes: String?
+        ) {
+            self.incidentIdentifier = incidentIdentifier
+            self.processName = processName
+            self.parentProcess = parentProcess
+            self.path = path
+            self.codeType = codeType
+            self.exceptionType = exceptionType
+            self.exceptionCodes = exceptionCodes
+        }
+
+        enum CodingKeys: String, CodingKey {
+            case incidentIdentifier = "incident_identifier"
+            case processName = "process"
+            case parentProcess = "parent_process"
+            case path = "path"
+            case codeType = "code_type"
+            case exceptionType = "exception_type"
+            case exceptionCodes = "exception_codes"
+        }
+    }
+
     /// The date of the crash occurrence.
     internal let date: Date?
     /// Crash report type - used to group similar crash reports.
+    /// In Datadog Error Tracking this corresponds to `error.type`.
     internal let type: String
     /// Crash report message - if possible, it should provide additional troubleshooting information in addition to the crash type.
+    /// In Datadog Error Tracking this corresponds to `error.message`.
     internal let message: String
-    /// Unsymbolicated stack trace of the crash.
-    internal let stackTrace: String
+    /// Unsymbolicated stack trace related to the crash (this can be either uncaugh exception backtrace or stack trace of the halted thread).
+    /// In Datadog Error Tracking this corresponds to `error.stack`.
+    internal let stack: String
+    /// All threads running in the process.
+    internal let threads: [Thread]
+    /// List of binary images referenced from all stack traces.
+    internal let binaryImages: [BinaryImage]
+    /// Meta information about the crash and process.
+    internal let meta: Meta
+    /// If any stack trace information was truncated due to crash report minimization.
+    internal let wasTruncated: Bool
```

Comment:
What happens if it gets truncated ? Where you will send the `stacktrace` continuation or you will not send it at all ?

Reply:
Truncation is added in https://github.com/DataDog/dd-sdk-ios/pull/538, we remove less meaningful frames and replace them with `...`, for example:

<img width="795" alt="124959128-3140c800-e01b-11eb-86df-03053de89f71" src="https://user-images.githubusercontent.com/2358722/125052453-457cd780-e0a4-11eb-8a56-08dd6740d3fc.png">